### ISSUE TITLE
Reader.query_periodic may be None in Reader.close()

### ISSUE THREAD

**Issue by mpe** [bitly/pynsq#122]
If there is no lookupd configured then ```Reader.query_periodic``` will be ```None```, leading to:

```
File "/usr/local/lib/python2.7/dist-packages/nsq/reader.py", line 265, in close
     self.query_periodic.stop()
AttributeError: 'NoneType' object has no attribute 'stop'
```

**Comment by mreiferson** [comment]
Hi @mpe - thanks for the report.  Any interest in submitting a fix?

**Comment by mpe** [comment]
I'll try and get you one this week.

**Comment by mreiferson** [comment]
fixed in #123
